src/component/title/mod.rs

```diff
@@ -307,3 +351,51 @@ impl Component {
         }
     }
 }
+
+// FIXME: Turn this into a generator once that works on stable Rust.
+fn abbreviate_category(category: Cow<'_, str>) -> Vec<Box<str>> {
```

Comment:
Are you sure that this method should belong here instead of expanding the scope of `livesplit-title-abbreviations` and putting it there?